src/views/Cookies/cookies.en.md

```diff
@@ -0,0 +1,45 @@
+# Cookie files policy and operational data
```

Comment:
Make sure this one can be translated via normal gettext flow.

Reply:
@Komzpa As far as I know we have no plans to translate the terms, I will clarify this point once again